canonical-web-and-design/maas.io
# Only load all the YouTube scripts when the user watches a video

## Done
Reduce the initial load of the site by limiting the YouTube scripts to only load when the video is played.

Page size down from ~3MB to ~1.5MB.

## QA
- Load the homepage
- Click the video in the hero of the site
- See the video loads and plays

## Issue / Card
Fixes https://github.com/canonical-web-and-design/maas.io/issues/517
